### PR TITLE
fix(shipper): added Walmart delivering sensor. Added new subject to Walmart delivered.

### PR DESCRIPTION
**Pull request by skewll**
Added Walmart Delivering Sensor

Updated Walmart Delivered Sensor subject array.
Updated Walmart Delivered tracking pattern.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests
- [ ] Documentation update
- [ ] Adds a new shipper
- [x] Update existing shipper

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:

### CODE REVIEW

File: custom_components/mail_and_packages/const.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 DOMAIN = "mail_and_packages"
 DOMAIN_DATA = f"{DOMAIN}_data"
-VERSION = "0.0.0-dev"  # Now updated by release workflow
+VERSION = "0.3.23"
```
Comment (firstof9) [inline custom_components/mail_and_packages/const.py:16]:
```suggestion
VERSION = "0.0.0-dev"  # Now updated by release workflow
```
This get's updated when we do a release no need to adjust this.

**Review by firstof9 — CHANGES_REQUESTED** [review]

**Comment by firstof9** [comment]
Just the recommendation and then formatting with black should check all the boxes :+1:

**Comment by skewll** [comment]
I didn't change the version at all. I'm not sure how that happened. I've pushed a new commit with the suggested version change. I also used Microsoft's black extension. I hope that's the right extension.

**Review by firstof9 — APPROVED** [review]
Looks good, pending tests.

**Comment by firstof9** [comment]
Thanks!

**Pull request merged**